homeassistant/components/velbus/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration does not poll.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
```

Comment:
Hmm, hos do you suggest we do this? As i'm not sure how it can be done as we only have 2 parameters, a port and a name.

The velbus system is not providing us with any names

Reply:
Does an average user have multiple of these systems at home? If not, we can just set the name to `Velbus` and otherwise you can set the port as the name.

Reply:
no, in most cases not